.yarnrc.yml

```diff
@@ -0,0 +1,5 @@
+enableTransparentWorkspaces: false
+
+nodeLinker: node-modules
```

Comment:
This disables PnP, because it doesn't work with Flow.

Reply:
Minds linking the upstream issue here?
```
https://github.com/yarnpkg/berry/issues/634
```